client/store-management-links/quick-link/index.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import React from '@wordpress/element';
+import { external, Icon } from '@wordpress/icons';
+import { __experimentalText as Text } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+export const QuickLink = ( { icon, title, href, isExternal, onClick } ) => {
+	return (
+		<div className="woocommerce-quick-links__item">
+			<a
```

Comment:
Should this use the `Link` component from `@woocommerce/components`? That would allow for navigating to other wc-admin powered pages without a full page load (if passed the `type` prop). Alternate idea: should we just update the `Link` component to be this `QuickLink` implementation?

Reply:
@jeffstieler thats a good point. My main concern would be styling but we could probably override it with css. 🤔 

I had forgotten that `Link` worked in that way so it *could* make sense to use that here. I'll investigate this.

Reply:
I've gone with the `Link` component here and it works well. 🎉 thanks this was a definite improvement. https://github.com/woocommerce/woocommerce-admin/pull/5476/commits/24e2171719d231dcd39305af5ff443a891f509ea